pump/storage/storage.go

```diff
@@ -1114,9 +1115,15 @@ func (a *Append) PullCommitBinlog(ctx context.Context, last int64) <-chan []byte
 
 		for {
 			startTS := last + 1
+			limitTS := atomic.LoadInt64(&a.maxCommitTS) + 1
+			if startTS > limitTS {
```

Comment:
just set `limitTS  to be startTS + 1 if  startTS >= limitTS` to by pass the bug
don't log warn log and sleep 10 second

Reply:
will it cause some bug if query some binlog is greater than the max commit ts?

Reply:
add a comment

Reply:
correct:
set  limitTS to be startTS  if startTS > limitTS if the bug about goleveldb will only trigger when
`start > limit`

[startTS,   limitTS( = startTS)     will get no any binlog